zmq.hpp

```diff
@@ -2653,6 +2653,15 @@ template<typename T = no_user_data> class poller_t
         throw error_t();
     }
 
+#if ZMQ_VERSION >= ZMQ_MAKE_VERSION(4, 3, 3)
+    size_t size() const noexcept
+    {
+        int rc = zmq_poller_size(const_cast<void *>(poller_ptr.get()));
+        ZMQ_ASSERT(rc >= 0);
+        return static_cast<size_t>(std::max(rc, 0));
```

Comment:
`ZMQ_ASSERT` always crashes on failure. The `std::max` can therefore be removed.

Reply:
Ah no, I confused that with libzmq's zmq_assert...